README.md

```diff
@@ -1,33 +1,45 @@
-# 25PieceCubePuzzle
-Instructions:
-clone, make, and run `./puzzle` and view results in `output.txt`
-The output file is as though, once the cube was assembled, it were sliced into 5 levels then laid out from bottom to top. The 
-numbers in each cell represent the pieces of each wooden block found on that layer. Try and visiualize all 5 layers immediately ontop of one another.
+# CubicConundrum
```

Comment:
If you actually go for this name, I'd probably change the name of your repository itself as well, and update your repo description to match whatever tagline you end up choosing.